tensorflow/core/kernels/data/dataset_test_base.h

```diff
@@ -359,8 +357,20 @@ class ConcatenateDatasetParams : public DatasetParams {
 
 template <typename T>
 struct GetNextTestCase {
+  GetNextTestCase(T dataset_params, std::vector<Tensor> expected_outputs)
+      : dataset_params(std::move(dataset_params)),
+        expected_outputs(std::move(expected_outputs)),
+        compare_order(true) {}
+
+  GetNextTestCase(T dataset_params, std::vector<Tensor> expected_outputs,
+                  bool compare_order)
```

Comment:
Use default value to avoid the need for the other constructor:
```
  GetNextTestCase(T dataset_params, std::vector<Tensor> expected_outputs,
                  bool compare_order = true)
```